x-pack/test/api_integration/apis/ml/saved_objects/update_trained_model_spaces.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+  const spacesService = getService('spaces');
+  const supertest = getService('supertestWithoutAuth');
+
+  const trainedModelId = 'trained_model';
+  const idSpace1 = 'space1';
+  const idSpace2 = 'space2';
+  const defaultSpaceId = 'default';
+
+  async function runRequest(
+    requestBody: {
+      modelIds: string[];
+      spacesToAdd: string[];
+      spacesToRemove: string[];
+    },
+    expectedStatusCode: number,
+    user: USER
+  ) {
+    const { body, status } = await supertest
+      .post(`/api/ml/saved_objects/update_trained_models_spaces`)
+      .auth(user, ml.securityCommon.getPasswordForUser(user))
+      .set(COMMON_REQUEST_HEADERS)
+      .send(requestBody);
+    ml.api.assertResponseStatusCode(expectedStatusCode, status, body);
+
+    return body;
+  }
+
+  describe('POST saved_objects/update_trained_models_spaces', () => {
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/ihp_outlier');
```

Comment:
AFAICS, we don't need this archive here.

Reply:
removed in https://github.com/elastic/kibana/pull/150901/commits/6b5f986e9f53b26c2b6ac60109f21e1752d09ab3